frontend/front/src/pages/Admin/assignment/AssignTable.js

```diff
@@ -175,6 +175,58 @@ const AssignTable = () => {
         }`;
       },
     },
+    {
```

Comment:
It looks like here you're adding new surveyor/completed columns instead of editing the existing ones.

Reply:
I don't recall this, I think I took the Button code from here. I might have saved, and prettier formatted this? I tried to see what was "deleted', but it's not showing me that.

Reply:
In that case could you remove this?

Reply:
Ok, removed. 

I think I know how I mistakenly put this there. I have an upstream clone now. Though I made it for a different reason, it should help me be more careful.